dbscripts/xml/upgrade/2.4.8-1_editorial_draft.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * 2.4.8-1_editorial_draft.xml
+  *
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * 2.4.8-1 editorial draft table creation.
+  *
+  -->
+
+<data>
```

Comment:
Table schemas should be written per `lib/pkp/dtd/xmlSchema.dtd`, not with explicit `CREATE TABLE` statements.